configure-basemap-style-parameters/src/main/java/com/esri/arcgismaps/sample/configurebasemapstyleparameters/screens/MainScreen.kt

```diff
@@ -0,0 +1,262 @@
+                contentAlignment = Alignment.Center
+            ) {
+                MapView(
+                    modifier = Modifier
+                        .fillMaxSize(),
+                    arcGISMap = mapViewModel.map
+                )
+                // show the "Show controls" button only when the bottom sheet is not visible
+                if (!bottomSheetState.bottomSheetState.isVisible) {
+                    Button(
+                        modifier = Modifier
+                            .align(Alignment.BottomCenter)
+                            .padding(bottom = 24.dp),
+                        onClick = {
+                            bottomSheetScope.launch {
+                                bottomSheetState.bottomSheetState.expand()
+                            }
+                        },
+                    ) {
+                        Text(
+                            text = "Show controls"
+                        )
+                    }
+                }
+                // constrain the bottom sheet to a maximum width of 380dp
+                Box(
+                    modifier = Modifier
+                        .widthIn(0.dp, 380.dp)
+                ) {
+                    BottomSheetScaffold(
+                        scaffoldState = bottomSheetState,
+                        sheetContent = {
+                            BoxWithConstraints(
+                                // constrain the height of the bottom sheet to 160dp
+                                Modifier
+                                    .heightIn(max = 160.dp)
+                                    .padding(8.dp)
+                            ) {
+                                Row {
+                                    Column {
+                                        // UI for setting the language strategy
+                                        LanguageStrategyControls(
+                                            languageStrategyOptions = mapViewModel.languageStrategyOptions,
+                                            onLanguageStrategyChange = { languageStrategy ->
+                                                mapViewModel.updateLanguageStrategy(languageStrategy)
+                                            },
+                                            languageStrategy = mapViewModel.languageStrategy,
+                                            enabled = (mapViewModel.specificLanguage == "None")
+                                        )
+                                    }
+                                    Divider(
+                                        color = Color.LightGray,
+                                        modifier = Modifier
+                                            .padding(8.dp)
+                                            .fillMaxHeight()
+                                            .width(2.dp)
+                                    )
+                                    Column {
```

Comment:
You could skip using a divider and split the two columns down the middle: 

```suggestion
                                Row(horizontalArrangement = Arrangement.SpaceBetween) {
                                    Column(Modifier.weight(0.5f)) {
                                        // UI for setting the language strategy
                                        LanguageStrategyControls(
                                            languageStrategyOptions = mapViewModel.languageStrategyOptions,
                                            onLanguageStrategyChange = { languageStrategy ->
                                                mapViewModel.updateLanguageStrategy(languageStrategy)
                                            },
                                            languageStrategy = mapViewModel.languageStrategy,
                                            enabled = (mapViewModel.specificLanguage == "None")
                                        )
                                    }

                                    Column(Modifier.weight(0.5f)) {
```